lib/berkeley_library/location/world_cat/libraries_request.rb

```diff
@@ -1,55 +1,65 @@
 require 'nokogiri'
+require 'json'
+require 'jsonpath'
 require 'berkeley_library/util'
 require 'berkeley_library/location/oclc_number'
 require 'berkeley_library/location/world_cat/symbols'
+require 'berkeley_library/location/world_cat/oclc_auth'
 
 module BerkeleyLibrary
   module Location
     module WorldCat
-      # @see https://developer.api.oclc.org/wcv1#/Holdings
+      # @see https://developer.api.oclc.org/wcv2#/Member%20General%20Holdings/find-bib-holdings
       class LibrariesRequest
         include BerkeleyLibrary::Util
 
-        XPATH_INST_ID_VALS = '/holdings/holding/institutionIdentifier/value'.freeze
+        JPATH_INST_ID_VALS = '$.briefRecords[*].institutionHolding.briefHoldings[*].oclcSymbol'.freeze
 
-        attr_reader :oclc_number, :symbols
+        attr_reader :oclc_number, :symbols, :oclc_token
 
         def initialize(oclc_number, symbols: Symbols::ALL)
+          @oclc_token = OCLCAuth.instance
           @oclc_number = OCLCNumber.ensure_oclc_number!(oclc_number)
           @symbols = Symbols.ensure_valid!(symbols)
         end
 
         def uri
-          @uri ||= URIs.append(libraries_base_uri, URIs.path_escape(oclc_number))
+          @uri ||= URIs.append(libraries_base_uri)
```

Comment:
the `URIs.append` call isn't necessary now that the OCLC number is being passed as a query parameter 🙂

Reply:
(actually, I'm not sure `@uri` is necessary either, you could just use `libraries_base_uri` in `execute()`)